app/views/sessions/_form.html.erb

```diff
@@ -1,12 +1,12 @@
 <%= form_for :session, url: session_path do |form| %>
   <div class="text-field">
     <%= form.label :email %>
-    <%= form.text_field :email, type: 'email' %>
+    <%= form.text_field :email, type: 'email', class: "input-field" %>
   </div>
 
   <div class="password-field">
     <%= form.label :password %>
-    <%= form.password_field :password %>
+    <%= form.password_field :password, class: "input-field" %>
   </div>
 
   <div class="submit-field">
```

Comment:
Missed adding style to the button.

![imagen](https://user-images.githubusercontent.com/12243331/104506770-72e04100-55ab-11eb-8f32-571543728094.png)

Reply:
The goal of this issue is to write all the generic classes in a effort to standardize and reuse those as much as possible, but it is not the goal to change all of the instances in which those classes could be used, the refactor of styles using these classes must be issues for the next sprint